### DISCUSSION TITLE
Can't clone the repository using the command this project provided...

### DISCUSSION THREAD

**Discussion by FreeBSD-fan** [streamdeck-linux-gui/streamdeck-linux-gui#204] [Q&A]
git clone <https://github.com/streamdeck-linux-gui/streamdeck-linux-gui.git>
bash: syntax error near unexpected token `newline'

**Comment by coolapso** [comment]
sounds like you don't have git installed. Please use [streamcontroller](https://github.com/StreamController/StreamController), this project is not going to be maintained in favor of it. More details [here](https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/discussions/203)
